web/static/js/settings/ops-settings.js

```
@@ -0,0 +1,87 @@
+    input.dataset.inflight = '1';
+    fetch(bp + '/api/v1/settings', {
+      method: 'PUT',
+      headers: {
+        'Content-Type': 'application/json',
+        'X-CSRF-Token': csrfToken
+      },
+      body: JSON.stringify(body)
+    }).then(function (r) {
+      if (r.ok) {
+        if (statusId) {
+          var pill = document.getElementById(statusId);
```

Biome (2.5.1)

[error] This var should be declared at the root of the enclosing function.

(lint/correctness/noInnerDeclarations)